build.gradle.kts

```diff
@@ -182,7 +191,7 @@ dependencies {
     testImplementation(extra["robolectric"] as String)
 }
 
-kapt {
+configure<org.jetbrains.kotlin.gradle.plugin.KaptExtension> {
```

Comment:
@eskatos Can't use `kapt` anymore?

Reply:
Not if you are not using the `plugins {}` block.
See https://docs.gradle.org/current/userguide/kotlin_dsl.html#type-safe-accessors